### PR TITLE
Fix build suggestions that will become warnings next month

### PR DESCRIPTION
**Pull request by Youssef1313**
image

### CODE REVIEW

File: docs/framework/windows-workflow-foundation/exceptions.md [hunk 1]
```diff
@@ -3,7 +3,8 @@ title: "Exceptions"
 ms.date: "03/30/2017"
 ms.assetid: 065205cc-52dd-4f30-9578-b17d8d113136
 ---
-# Exceptions
+# Exceptions <!-- TODO: I'm unfamiliar with this area. What should the title be updated to? -->
```
Comment (IEvangelist) [inline docs/framework/windows-workflow-foundation/exceptions.md:6]:
```suggestion
# Exceptions in Workflows
```

File: docs/framework/windows-workflow-foundation/expressions.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ title: "Expressions - WF"
 ms.date: "03/30/2017"
 ms.assetid: c42341a9-43a1-462c-bffb-c5de004aa428
 ---
-# Expressions
+# Expressions <!-- TODO -->
```
Comment (IEvangelist) [inline docs/framework/windows-workflow-foundation/expressions.md:6]:
```suggestion
# Expressions in Workflows
```

File: docs/framework/windows-workflow-foundation/security.md [hunk 3]
```diff
@@ -3,7 +3,8 @@ title: "Security"
 ms.date: "03/30/2017"
 ms.assetid: 737ec121-bfc5-4b75-a504-2d53c2c8af39
 ---
-# Security
+# Security <!-- TODO -->
```
Comment (IEvangelist) [inline docs/framework/windows-workflow-foundation/security.md:6]:
```suggestion
# Workflow security
```

File: docs/csharp/language-reference/compiler-options/deterministic-compiler-option.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ helpviewer_keywords:
   - "deterministic compiler option [C#]"
   - "/deterministic compiler option [C#]"
 ---
-# -deterministic
+# -deterministic compiler option (C#)
```
Comment (Youssef1313) [inline docs/csharp/language-reference/compiler-options/deterministic-compiler-option.md:11]: Do you want me to add this suffix to all compiler options for consistency?

File: docs/csharp/tour-of-csharp/arrays.md [hunk 5]
```diff
@@ -5,7 +5,7 @@ ms.date: 02/27/2020
 ms.assetid: a440704c-9e88-4c75-97dd-bfe30ca0fb97
 ---
 
-# Arrays
+# Arrays in C\#
```
Comment (Youssef1313) [inline docs/csharp/tour-of-csharp/arrays.md:8]: Do you want me to add this suffix to all articles in *tour-of-csharp* for consistency?

File: docs/fsharp/language-reference/arrays.md [hunk 6]
```diff
@@ -3,7 +3,7 @@ title: Arrays
 description: Learn how to create and use arrays in the F# programming language.
 ms.date: 05/16/2016
 ---
-# Arrays
+# Arrays in F\#
```
Comment (Youssef1313) [inline docs/fsharp/language-reference/arrays.md:6]: Do you want me to add this suffix to all articles in F# reference for consistency?

File: docs/standard/design-guidelines/arrays.md [hunk 7]
```diff
@@ -8,7 +8,8 @@ helpviewer_keywords:
   - "empty arrays"
 ms.assetid: 66a1b3d8-6f3f-4715-b235-e1ff95e32d8e
 ---
-# Arrays
+# Arrays design guidelines
```
Comment (Youssef1313) [inline docs/standard/design-guidelines/arrays.md:11]: Do you want me to add this suffix to all design guidelines articles for consistency?

File: docs/visual-basic/reference/command-line-compiler/deterministic.md [hunk 8]
```diff
@@ -6,7 +6,7 @@ helpviewer_keywords:
   - "-deterministic compiler option [Visual Basic]"
   - "-deterministic compiler option [Visual Basic]"
 ---
-# -deterministic
+# -deterministic compiler option (Visual Basic)
```
Comment (Youssef1313) [inline docs/visual-basic/reference/command-line-compiler/deterministic.md:9]: Do you want me to add this suffix to all VB compiler options for consistency?

File: docs/csharp/language-reference/compiler-options/deterministic-compiler-option.md (hunk 4 above)
Comment (BillWagner) [reply docs/csharp/language-reference/compiler-options/deterministic-compiler-option.md:11]: yes

File: docs/csharp/tour-of-csharp/arrays.md (hunk 5 above)
Comment (BillWagner) [reply docs/csharp/tour-of-csharp/arrays.md:8]: No, I don't think this is common enough.

File: docs/fsharp/language-reference/arrays.md (hunk 6 above)
Comment (BillWagner) [reply docs/fsharp/language-reference/arrays.md:6]: I don't think so, but pinging @cartermp for his opinion.

File: docs/visual-basic/reference/command-line-compiler/deterministic.md (hunk 8 above)
Comment (BillWagner) [reply docs/visual-basic/reference/command-line-compiler/deterministic.md:9]: yes

File: docs/standard/design-guidelines/arrays.md (hunk 7 above)
Comment (BillWagner) [reply docs/standard/design-guidelines/arrays.md:11]: I think no, because this is copyrighted content.

**Comment by BillWagner** [comment]
@Youssef1313 Is this something you still want to work on? If not, let's close it.

File: docs/fsharp/language-reference/arrays.md (hunk 6 above)
Comment (cartermp) [reply docs/fsharp/language-reference/arrays.md:6]: Sure!

**Comment by Youssef1313** [comment]
Given that a few months passed and it didn't become warnings, I'm going to close.

**Pull request closed without merging**